CODING_STANDARDS.md

```diff
@@ -9,11 +9,11 @@ rewritten to comply with these rules.
 
 1. Document your code in source files and the manual. (tm)
 
-1. PHP is implemented in C99.  The optional fixed-width integers from
+2. PHP is implemented in C99.  The optional fixed-width integers from
```

Comment:
The numbering looks intentional. Markdown allows using any number for numbered lists, but the output will still be sequential starting from 1. This avoids changing all lines when an element is added in the middle.